src/libasr/pass/array_op.cpp

```diff
@@ -310,11 +327,6 @@ class ReplaceArrayOp: public ASR::BaseExprReplacer<ReplaceArrayOp> {
                             al, loc, left, (ASR::binopType)x->m_op,
                             right, x->m_type, nullptr));
 
-            case ASR::exprType::UnsignedIntegerBinOp:
```

Comment:
To be reverted: https://github.com/lcompilers/lpython/pull/1775/files#diff-51616d9e7a321bfa6245084060933b377e1ab9e391b2b4f248fd9d8ee9bf2ce0